bootstrap-salt.sh

```diff
@@ -4810,6 +4836,7 @@ install_opensuse_stable_deps() {
     fi
 
     # Is the repository already known
+    # FIXME: cleartext download over unsecure protocol (HTTP)
```

Comment:
Oh thanks for telling me about that `rpm` feature, I have not used it in a while and I was assuming it wouldn't do the same as Debian/RHEL regarding verifying GPG signatures.

Yes in that case securing the GPG public key will be enough.